library/Makefile

```diff
@@ -138,7 +137,7 @@ ifneq ($(APPLE_BUILD),0)
 endif
 endif
 
-libmbedx509.$(SOEXT_X509): $(OBJS_X509) $(CRYPTO)libmbedcrypto.so
```

Comment:
Seems that with LOCAL_LDFLAGS set to the crypto repo, this dependency can be obtained and linked with, which is good.

Is `$(CRYPTO)` needed in any other makefiles?

Reply:
No, but they define their own copy of it, so I'll unify it.